os/include/tinyara/binary_manager.h

```diff
@@ -224,11 +215,11 @@ struct binmgr_request_s {
 };
 typedef struct binmgr_request_s binmgr_request_t;
 
-struct binmgr_createbin_response_s {
+struct binmgr_getdownloadpath_response_s {
 	int result;
 	char binpath[BINARY_PATH_LEN];
 };
-typedef struct binmgr_createbin_response_s binmgr_createbin_response_t;
+typedef struct binmgr_getdownloadpath_response_s binmgr_getdownloadpath_response_t;
```

Comment:
I think it is too long.
How about `bingmr_getpath_response_t`?

Reply:
I updated as you said.